product_docs/docs/eprs/7/07_common_operations/08_replicating_ddl_changes/01_ddl_change_replication.mdx

```diff
@@ -20,4 +20,8 @@ The following is the series of steps that occur during the DDL change replicatio
 -   The DDL statement is executed against the target table. The replication triggers and shadow table are modified accordingly. If an error occurs, you're notified and the operation stops. If `ddlChangeTableLock` is set to `true`, the exclusive lock is released.
 -   These two actions are repeated on the target table for each database in the replication system.
 -   The in-memory table metadata definition is refreshed to reflect the DDL change. You're notified when the operation completes successfully.
--   If an error occurs during these steps, any changes up to that point are rolled back so that the publication table, replication triggers, and shadow table revert to their original state prior to the start of this operation. If one or more databases goes down before the operation finishes, the publication is marked as dirty to avoid further replication events.
\ No newline at end of file
+-   If an error occurs during these steps, any changes up to that point are rolled back so that the publication table, replication triggers, and shadow table revert to their original state prior to the start of this operation. If one or more databases goes down before the operation finishes, the publication is marked as dirty to avoid further replication events.
+
+!!! Note
+   When you execute the alterDDL command, non-MDN nodes usually don't have any CDC changes during the MMR setup. If there are any changes on the altered table in the non-MDN node, it can result in data loss and a break in replication. 
```

Comment:
Please update the note as below

When you execute the alterDDL command, non-MDN nodes should not have any CDC changes during that window for MMR setup. If there are any changes on the altered table during that window in the non-mdn nodes, it may result in data loss and a break in replication.